projects/components/src/data-exporter/data-exporter.component.spec.ts

```diff
@@ -48,17 +48,33 @@ describe('DataExporterColumnsWithoutDisplayName', () => {
                 ...TestData.exportDataWithoutDisplayName.map(row => Object.values(row)),
             ];
             const csvString = downloadService.createCsv(exportData);
-            expect(downloadService.downloadCsvFile).toHaveBeenCalledWith(
-                csvString,
-                exporter.component.fileName
-            );
+            expect(downloadService.downloadCsvFile).toHaveBeenCalledWith(csvString, exporter.component.fileName);
             this.finder.detectChanges();
             expect(this.finder.hostComponent.dataExporterOpen).toBe(false);
             done();
         });
 
         exporter.clickExport();
     });
+
+    it('displays field name when there is no display name', function(this: TestExporterColumnsWithoutDisplayNameFinder, done): void {
+        const exporter = this.finder.find(DataExporterWidgetObject);
+        let dropdownMenu = document.querySelector('clr-dropdown-menu');
+        expect(dropdownMenu).toBeNull();
+        exporter.component.selectAllControl.setValue(false);
+        this.finder.detectChanges();
+        dropdownMenu = document.querySelector('clr-dropdown-menu');
+        const selections = dropdownMenu.querySelectorAll('li');
+        expect(selections.length).toEqual(2);
+        expect(selections[0].querySelector('label').innerText.trim()).toEqual('col1');
```

Comment:
We don't make queries from jasmine, all queries should be hidden behind the widget wrapper, you are duplicating code from the widget wrapper